Code/Framework/AzCore/AzCore/DOM/DomBackendRegistry.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/DOM/DomBackendRegistry.h>
+
+#include <AzCore/Interface/Interface.h>
+
+namespace AZ::DOM
+{
+    BackendRegistry* BackendRegistry::s_instance = nullptr;
```

Comment:
According to the C++ style guide static variables should start with a capital letter so this should be `Instance`.